README.md

```diff
@@ -2,17 +2,22 @@
 
 The new home of the betanextbest.wellcomecollection.org website.
 
-# client
-
-    cd client
-    npm install
-    npm run compile OR npm run compile:watch (for developing)
-       
-# server
-    
-    cd server
-    npm install
-    npm run watch
-    
+## client
+
+    - `cd client`
```

Comment:
I would however suggest for the others that have no options, that they are codeblocks and shouldn't be lists.